### PR TITLE
Create pull request for adding helm index to main branch

### PR DESCRIPTION
**Pull request by andriisoldatenko**
## Description

Please include the following:

This PR introduce new release step: Create pull request for adding helm index to main branch 

## How can this be tested?

You can test it after merging or together with ->

related to https://github.com/Dynatrace/dynatrace-operator/pull/2078

## Checklist

- [x] Unit tests have been updated/added
- [x] PR is labeled accordingly with a single label
- [x] I have read and understood the [contribution guidelines](https://github.com/Dynatrace/dynatrace-operator/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Comment by andriisoldatenko** [comment]
tested in my fork and it worked https://github.com/andriisoldatenko/dynatrace-operator/pull/47

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -319,7 +320,30 @@ jobs:
...
           draft: true
           fail_on_unmatched_files: true
-
+      - name: Update index helm file
+        uses: actions/checkout@v3
+        run: |
+          echo "Updating Helm repo index"
+          ./hack/build/ci/generate-new-helm-index-yaml.sh
+      - name: Create pull request for adding helm index to main branch
+        uses: peter-evans/create-pull-request@v5
```
Comment (luhi-DT) [inline .github/workflows/release.yaml:329]: is this action the way to go for creating PRs? looks like it comes out of a private repo

**Review by luhi-DT — APPROVED** [review]
LGTM

File: .github/workflows/release.yaml (hunk 1 above)
Comment (andriisoldatenko) [reply .github/workflows/release.yaml:329]: Repo is public https://github.com/peter-evans/create-pull-request

**Pull request merged**